packages/backend/index.js

```diff
@@ -7,9 +7,15 @@ const bodyParser = require("body-parser");
 // Firebase set up
 const firebaseAdmin = require("firebase-admin");
 const firebaseServiceAccount = require("./firebaseServiceAccountKey.json");
-const { userOnly, adminOnly } = require("./middlewares/auth");
+const { withAddress } = require("./middlewares/auth");
```

Comment:
Another idea for this middleware name: `withProviderAddress`

Reply:
Provider sounds a bit too ambiguous to me. What about `withConnectedAddress` or `withWalletAddress`?

Reply:
Proposed «provider» since it's how is called on the frontend (userProvider, injectedProvider, localProvider). But I like `withConnectedAddress` also.